collect_app/src/main/res/values/strings.xml

```diff
@@ -13,6 +13,7 @@
 <resources xmlns:tools="http://schemas.android.com/tools" tools:locale="en"
            tools:ignore="MissingTranslation">
 <string name="accept_location">Record Location</string>
+<string name="show">Show password</string>
```

Comment:
As @lognaturel mentioned [here](https://github.com/opendatakit/collect/pull/551#discussion_r105174512), can you please change it to `show_password`. :)